packages/ai/README.md

```diff
@@ -8,10 +8,15 @@ To learn more about how to use the AI SDK, check out our [API Reference](https:/
 
 ## Installation
 
-You will need Node.js 18+ and pnpm installed on your local development machine.
+Install on your local development machine:
+
+- Node.js `v18` or `v20`. We recommend managing Node installation and versions with [fnm](https://github.com/Schniz/fnm) or [nvm](https://github.com/nvm-sh/nvm).
+- For package management we recommend (and the subsequent docs assume) [pnpm](https://pnpm.io/installation) `v8`, e.g. installed via `npm install -g pnpm@8.6.9`.
+
+Once you've got those in place:
 
 ```shell
-npm install ai
```

Comment:
NPM is fine for using the AI SDK ( as is pnpm, bun, etc) - only NPM here for illustration purposes.